src/sentry/services/hybrid_cloud/organization_mapping/impl.py

```diff
@@ -22,18 +23,20 @@ def create(
         idempotency_key: Optional[str] = "",
         # There's only a customer_id when updating an org slug
         customer_id: Optional[str] = None,
-        user: Optional[int] = None,
+        status: Optional[OrganizationStatus] = None,
     ) -> RpcOrganizationMapping:
 
         if idempotency_key:
             org_mapping, _created = OrganizationMapping.objects.update_or_create(
                 slug=slug,
                 idempotency_key=idempotency_key,
                 region_name=region_name,
+                verified=False,
```

Comment:
I ended up adding this check to ensure we only attempt an upsert for new, unverified mappings instead of potentially overwriting a valid existing mapping if for some reason we pass a matching idempotency key (e.g. we supply "" in an update slug case)